content/docs/command-reference/run.md

```diff
@@ -270,15 +270,19 @@ $ dvc run -n second_stage './another_script.sh $MYENVVAR'
   affect any DVC operations.
 
 - `--live <path>` - specify the directory `path` for
-  [Dvclive](/doc/dvclive/dvclive-with-dvc) to write logs in. `path` will be
-  tracked (<abbr>cached</abbr>) by DVC. Saved in the `live` field of `dvc.yaml`.
+  [DVCLive](/doc/dvclive/user-guide/dvclive-with-dvc) to write logs in. `path`
+  will be tracked (<abbr>cached</abbr>) by DVC. Saved in the `live` field of
+  `dvc.yaml`.
 
-- `--live-no-cache <path>` - the same as `-o` except that the `path` is not
+- `--live-no-cache <path>` - the same as `--live` except that the `path` is not
   tracked by DVC. Useful if you prefer to track it with Git.
 
-- `--live-no-summary` - passes `summary=False` to Dvclive config.
+- `--live-no-summary` - deactivates DVCLive
```

Comment:
This is not strictly related to the html change but I think it just make sense to be consistent